cfme/tests/configure/test_landing_page.py

```diff
@@ -131,7 +135,11 @@ def set_to_default(page, my_settings):
     view.tabs.visual.save.click()
 
 
-@pytest.mark.parametrize('start_page', LANDING_PAGES, scope="module")
+@pytest.mark.parametrize(
+    "start_page",
+    LANDING_PAGES if current_version() > "5.10" else LANDING_PAGES_5_9,
```

Comment:
Please do not use current_version, use a lambda expression here to use `appliance` from the global fixtures.

Reply:
I know, I have been trying to resolve that, but couldn't find a good workaround. One workaround that I found was to use `uncollectif` along with parametrization. It is something like this:
Keeping LANDING_PAGES as it is, something like this worked well.
```
LANDING_PAGES_5_9= [
    "Compute / Infrastructure / Requests",
    "Middleware / Providers",
    "Middleware / Domains",
    "Middleware / Servers",
    "Middleware / Deployments",
    "Middleware / Datasources",
    "Middleware / Messagings",
    "Middleware / Topology",
]


@pytest.mark.uncollectif(
lambda appliance, start_page: start_page in LANDING_PAGES_5_9 and appliance.version > "5.10",
reason="Page does not exist on 5.10")
@pytest.mark.parametrize("start_page", LANDING_PAGES, scope="module")
def test_landing_page_admin(start_page, appliance, my_settings, request):
    """
            This test checks the functioning of the landing page; 'Start at Login'
            option on 'Visual' tab of setting page for administrator. This test case doesn't
            check the exact page but verifies that all the landing page options works properly.
    """
    request.addfinalizer(lambda: set_to_default('Cloud Intel / Dashboard', my_settings))
    assert set_landing_page(start_page, appliance, my_settings)

```

Let me know what you think. And sorry I didn't ack your comments earlier.

Reply:
@valaparthvi thanks for continuing to work through it!! Well done. I think I do like that uncollect syntax as a way to not use current_appliance. Its pretty clear and concise.

Reply:
Feel free to open an rc-regression PR if you would like to change that in this release, or we can spend some time discussion options.